Specs/DateTime/Dutch/TimePeriodParser.json

```diff
@@ -1048,20 +1019,19 @@
           }
         },
         "Start": 43,
-        "Length": 18
+        "Length": 19
       }
     ]
   },
   {
-    "Input": "Leg meeting vast van 13:30 tot 3:30",
+    "Input": "Leg meeting vast van 1:30 's middags tot 3:30",
```

Comment:
Same as previous.

Reply:
added spec.
without "morning", the case is parsed as (T13:30,T03:30,PT14H) instead of (T13:30,T15:30,PT2H). 
Same happens in English with "from 13:30 to 3:30"